src/test/java/com/beust/jcommander/UnixStyleUsageFormatterTest.java

```diff
@@ -11,6 +11,16 @@
 @Test
 public class UnixStyleUsageFormatterTest {
 
+    /**
+     * Returns a resource bundle for the en_US locale, this is to prevent needing an exhaustive list of locales
+     * to allow the related tests to run on foreign machines.
+     *
+     * @return a hard-coded resource bundle for the en_US locale
+     */
+    private static ResourceBundle getResourceBundle() {
```

Comment:
Duplicated from `DefaultUsageFormatterTest`. Let's put this static method in a single location and call it there.